app/styles/mobile.scss

```diff
@@ -69,8 +69,8 @@
   padding-top: 50px;
   text-align: center;
   background-color: $black;
-  a{
-    color:$white;
+  a:hover{
+    color:purple;
```

Comment:
Color `purple` should be written in hexadecimal form as `#800080`<br>Color literals like `purple` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space